libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/AccessibilityUtils.java

```diff
@@ -26,4 +33,22 @@ public static int getSnackbarDuration(Context ctx, int defaultDuration) {
         return defaultDuration == Snackbar.LENGTH_INDEFINITE ? Snackbar.LENGTH_INDEFINITE
                 : isAccessibilityEnabled(ctx) ? SNACKBAR_WITH_ACTION_DURATION_IN_MILLIS : defaultDuration;
     }
+
+    public static void disableHintAnnouncement(@NonNull TextView textView) {
+        ViewCompat.setAccessibilityDelegate(textView, new AccessibilityDelegateCompat() {
+            @Override public void onInitializeAccessibilityNodeInfo(View host, AccessibilityNodeInfoCompat info) {
+                super.onInitializeAccessibilityNodeInfo(host, info);
+                info.setHintText(null);
+            }
+        });
+    }
+
+    public static void enableAccessibilityHeading(@NonNull View view) {
```

Comment:
NitPick: We might want to add a comment to this method that it can be removed and replaced with `android:accessibilityHeading="true"` when minSdk == 28. Wdyt?

Reply:
Awesome suggestion!

Reply:
Resolved in 76bcce4